chart/dashboard.yaml

```diff
@@ -0,0 +1,14 @@
+# dashboard.yaml
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRoute
+metadata:
+  name: traefik-dashboard
+spec:
+  entryPoints:
+    - web
+  routes:
+    - match: Host(`traefik.admin.dev.ubivius.tk`) && (PathPrefix(`/dashboard`) || PathPrefix(`/api`))
```

Comment:
Just to be consistent with the filebeat integration, would you like to add the identical part in values file?